components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -338,7 +338,16 @@ RewardsServiceImpl::~RewardsServiceImpl() {
   StopNotificationTimers();
 }
 
-void RewardsServiceImpl::ConnectionClosed() {
+void RewardsServiceImpl::OnProcessDisconnected() {
```

Comment:
Hey @zenparsing,
Would you mind withdrawing the `RewardsService` changes? Soon the profiles will share the `mojom::LedgerFactory` via a `static` local `mojo::Remote<>`. Also, no disconnect handler will be needed on the factory, only a call to `reset_on_disconnect()`.

Reply:
I'd like to understand better how crash handling will behave. Let's discuss next week.

Reply:
Sure! 👍🏻

Reply:
Removed from this PR - created https://github.com/brave/brave-browser/issues/30680 to address.